.github/workflows/validate-repackaged-binary-sizes.yml

```diff
@@ -23,21 +23,21 @@ jobs:
       fail-fast: false
       matrix:
         whl:
-          - url: https://download.pytorch.org/whl/test/cu117_pypi_cudnn/torch-1.13.1%2Bcu117.with.pypi.cudnn-cp310-cp310-linux_x86_64.whl
-            python: "3.10"  # python version to use for smoke tests
-            upload_artifact: false # upload the repackaged binary as an artifact
-          - url: https://download.pytorch.org/whl/test/cu117_pypi_cudnn/torch-1.13.1%2Bcu117.with.pypi.cudnn-cp37-cp37m-linux_x86_64.whl
-            python: "3.7"
+          - url: https://download.pytorch.org/whl/test/cu121/torch-2.2.0-cu121-cp312-cp312-linux_x86_64.whl
```

Comment:
Weird, the job is failing https://github.com/pytorch/builder/actions/runs/7647128523/job/20837378299?pr=1686 because of 403 Forbidden, the path looks fine I think

Reply:
Yes typo should be '+' after version not '-'